kolibri/plugins/learn/assets/src/my_downloads/views/MyDownloads.vue

```diff
@@ -139,6 +154,45 @@
           });
         }
       },
+      startPolling() {
+        this.pollingInterval = setInterval(async () => {
+          await this.fetchDownloads();
+          this.fetchDevices().then(devices => {
+            for (const device of devices) {
+              this.networkDevices.push(Object.assign(device));
```

Comment:
Every time you poll, your list of devices is going to get longer (with duplicates).

I think we could consolidate some of this logic somewhat across the different places that the learn `useDevices` composable is used, but might be good for us to cohack on that, as I've made some tweaks to this in the place that I suspect you copied this from.